packages/api/tests/lambdas/test-publish-reports.js

```diff
@@ -1,922 +0,0 @@
-    });
-  } finally {
-    // revert the mocking
-    executionPublishMock();
-    getExecutionHistoryStub.restore();
-  }
-});
-
-test.serial('handler publishes input to failed execution if failed step input cannot be retrieved', async (t) => {
```

Comment:
We should delete the fixtures that were used by the tests if we're removing the tests

Reply:
Fixed in 9847b3935